kustomize/commands/localize/localize.go

```diff
@@ -62,6 +68,26 @@ kustomize localize https://github.com/kubernetes-sigs/kustomize//api/krusty/test
 			if err != nil {
 				return errors.Wrap(err)
 			}
+
+			if !f.noVerify && f.scope == "" {
+				originalBuild, err := runBuildCmd(buildBuffer, buildCmd, args.target)
+				if err != nil {
+					return errors.Wrap(err)
+				}
+
+				localizedBuild, err := runBuildCmd(buildBuffer, buildCmd, dst)
+				if err != nil {
+					return errors.Wrap(err)
+				}
+
+				if localizedBuild == originalBuild {
+					log.Printf("VERIFICATION SUCCESS: `kustomize build` for %s and %s are the same after localization.", args.target, dst)
```

Comment:
minor: I think this log could benefit of having a line break at the end, since `Printf()` won't add it by default.